mixed/flink/flink-common/src/main/java/com/netease/arctic/flink/catalog/factories/FlinkCatalogFactory.java

```diff
@@ -70,31 +80,58 @@ public Catalog createCatalog(Context context) {
     final String defaultDatabase = helper.getOptions().get(DEFAULT_DATABASE);
     String metastoreUrl = helper.getOptions().get(ArcticCatalogFactoryOptions.METASTORE_URL);
 
+    String amoroCatalogName =
+        ArcticThriftUrl.parse(metastoreUrl, THRIFT_TABLE_SERVICE_NAME).catalogName();
+    UnifiedCatalog unifiedCatalog =
+        UnifiedCatalogLoader.loadUnifiedCatalog(metastoreUrl, amoroCatalogName, Maps.newHashMap());
+    Configuration hadoopConf = unifiedCatalog.authenticationContext().getConfiguration();
+
+    TableFormat catalogTableFormat =
+        TableFormat.valueOf(unifiedCatalog.properties().get(TABLE_FORMATS));
```

Comment:
We are willing to support multiple table formats in one catalog, so you can't assume that there is only one format here.